src/browser.ts

```diff
@@ -137,14 +137,13 @@ class Honeybadger extends Client {
         // eslint-disable-next-line @typescript-eslint/no-this-alias
         const client = this
         func.___hb = <WrappedFunc>function () {
-          const onErrorEnabled = client.config.enableUncaught
           // Catch if:
           //   1. We explicitly want to catch (i.e. if the error could be
           //      caught before reaching window.onerror)
           //   2. The browser provides less information via the window.onerror
           //      handler
           //   3. The window.onerror handler is unavailable
-          if (opts.catch || preferCatch || !onErrorEnabled) {
+          if (opts.catch || preferCatch || !client.config.enableUncaught) {
```

Comment:
This is confusing, but I think I understand it.
Basically, add the breadcrumbs if `opts.catch` or `preferCatch` but only call notify if `enableUncaught` is set.
What is the point of `opts.catch` if it cannot take precedence over the global config though (`enableUncaught`) ?

Reply:
That's a good question. The answer is that in the past we supported a `Honeybadger.wrap` method that shared this underlying implementation, and it used `opts.catch` to signal that it wants to explicitly report the error. I think I can actually remove `opts.catch` since `Honeybadger.wrap` was removed (it was a pain to support).